versions.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 5.11"
+      version = ">= 5.26"
```

Comment:
**Note:** Support for instance Maintenance policy was released in [5.26](https://github.com/hashicorp/terraform-provider-aws/pull/34430#issuecomment-1815493598)

Reply:
Thanks for pointing this out.